<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_estatico_sincuenta">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>620</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>¿Aun sin cuenta?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imagen_identificate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>721</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>recursos/recursos_usuario/image(4).png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contrasena">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgb(246, 245, 244);
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 13px;
    color: #606266;
    min-height: 18px;
}
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_telefono">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Teléfono:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_invitado">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>530</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: white;
	color:#00000;
	border:none;
	border-radius: 15px;
	font-weight: bold;
	font-size: 14px;
}
QPushButton:hover {
	background-color: rgb(222, 221, 218);
}

QPushButton:pressed {
	background-color: rgb(192, 191, 188);
}</string>
   </property>
   <property name="text">
    <string>Ingresa como invitado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_identificatetxt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Accede para comprar boletos y getionar tus reservaciones de forma mas sencilla.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_registrate">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: white;
	color:#1061C4;
	border: 2px solid #1061C4;
	border-radius: 15px;
	font-weight: bold;
	font-size: 15px;
}
QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
   </property>
   <property name="text">
    <string>regístrate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_identificate">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>¡Hola, identifícate!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_telefono">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgb(246, 245, 244);
    border: 1px solid #dcdfe6;
    border-radius: 6px;
    padding: 6px 12px;
    font-size: 13px;
    color: #606266;
    min-height: 18px;
}

</string>
   </property>
   <property name="inputMask">
    <string>9999999999</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estatico_contrasena">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>349</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Contraseña:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_continuar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>477</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 15px;
	font-weight: bold;
	font-size: 22px;
}
QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
